src/components/FormValidator.js

```diff
@@ -84,4 +85,12 @@ export default class FormValidator {
     });
     this._setEventListiners();
   }
+
+  resetValidation() {
+    const inputList = this._element.querySelectorAll(".modal__input");
+    inputList.forEach((input) => {
+      this._hideInputError(input);
+      this._toggleButtonState();
```

Comment:
`this._toggleButtonState();` should be called only **once** to toggle the button here rather than in the loop


```js

    this._toggleButtonState();  // once
    inputList.forEach((input) => {
      this._hideInputError(input);
    });
```

Reply:
Dear Gennady,
i solve the issue could you check for me please